ui-tests/src/test/resources/org/openmrs/uitestframework/test.properties

```diff
@@ -5,4 +5,5 @@ login.location=Pharmacy
 webdriver=chrome
 login.auto=false
 headless=true
-webdriver.gecko.driver=firefoxdriver/linux/geckodriver
\ No newline at end of file
+webdriver.gecko.driver=firefoxdriver/linux/geckodriver
+
```

Comment:
get out this un originally existing trailing space

Reply:
Resolved thanks